app/helpers/application_helper.rb

```diff
@@ -1055,6 +1055,12 @@ def javascript_flash
       render_to_string(:partial => "layouts/flash_msg")).for_render
   end
 
+  def open_window(url)
```

Comment:
Wondering if this should be `javascript_open_window` to keep in line with the others?